test/integration/serverless/run.sh

```diff
@@ -233,65 +239,71 @@ for function_name in "${all_functions[@]}"; do
                 perl -p -e "s/(architecture:)(x86_64|arm64)/\1XXX/g" |
                 perl -p -e "s/$stage/XXXXXX/g" |
                 perl -p -e "s/[ ]$//g" |
-                sort
```

Comment:
Will losing sorting introduce flakiness?

Reply:
I don't think so as it's now sorted in the recorder extension